### PR TITLE
[FAB-18489] fix links in 2.1 release docs

### PR DESCRIPTION
**Pull request by nikhil550**
Signed-off-by: Nikhil Gupta <ngupta@symbridge.com>

#### Type of change

- Bug fix
- Documentation update

#### Description

Fix links detected by the link checker in the release-2.1 docs

### CODE REVIEW

**Review by denali49 — DISMISSED** [review]
lgtm, thanks Nik.

File: docs/source/developapps/transactionhandler.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ shows you the exact form of these handlers.
 
 Once a handler has been added to the smart contract, it will be invoked during
 transaction processing. During processing, the handler receives `ctx`, the
-[transaction context](./transationcontext.md), performs some processing, and
+[transaction context](transationcontext.html), performs some processing, and
```
Comment (denali49) [inline docs/source/developapps/transactionhandler.md]: same typo as other one.  Don't know how I missed this one.

Comment (nikhil550) [reply docs/source/developapps/transactionhandler.md]: Thanks for finding. Fixed

**Review by denali49 — APPROVED** [review]
lgtm, thanks Nik.

**Comment by ryjones** [comment]
@denyeart it looks like this is tripping up on a go modules requirement. Should it be force-merged? https://github.com/hashicorp/hcl/issues/449

**Review by denyeart — APPROVED** [review]

**Comment by denyeart** [comment]
> @denyeart it looks like this is tripping up on a go modules requirement. Should it be force-merged? [hashicorp/hcl#449](https://github.com/hashicorp/hcl/issues/449)

Yeah, I'll force merge. Anybody know why this Go dependency fails on 2.0 and 2.1, but not 2.2 or 2.3?

**Pull request merged**